### PR TITLE
Refactored NodeIPEndpoint class

### PR DESCRIPTION
**Pull request by bmoo0**
address, tcpPort and udpPort members have been renamed with the prefix m_. They have also been made private and given accessor and mutator methods.

### CODE REVIEW

File: libp2p/Host.cpp [hunk 1]
```diff
@@ -257,7 +257,7 @@ void Host::startPeerSession(Public const& _id, RLP const& _rlp, unique_ptr<RLPXF
     }
     if (p->isOffline())
         p->m_lastConnected = std::chrono::system_clock::now();
-    p->endpoint.address = _s->remoteEndpoint().address();
+    p->endpoint.address() = _s->remoteEndpoint().address();
```
Comment (gumb0) [inline libp2p/Host.cpp]: Is this assigning to temporary? How is this possible?

**Comment by gumb0** [comment]
clang-format didn't reformat all your commits, try doing `git clang-format b0df983ebd7c1e6679e10930601f94fcec99eb48`
Sorry if the instructions for it are not clear

**Comment by bmoo0** [comment]
@gumb0 That should be fixed now, sorry I had some trouble with it initially but now I seem to have gotten it working.

**Comment by gumb0** [comment]
Looks good, could you squash the commits now?

**Comment by bmoo0** [comment]
All changes should be squashed into the latest commit now. Sorry about that.

**Comment by gumb0** [comment]
@bmoo0 Sorry, there were still 7 commits in this PR, there should have been 1 left after squashing. Trying to fix that I deleted older commits from your branch, then github automatically closed the PR (that was not what I wanted) Anyway here's another PR with your commit https://github.com/ethereum/cpp-ethereum/pull/5167

**Pull request closed without merging**